.github/workflows/belt-hash.yml

```diff
@@ -31,12 +31,10 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: RustCrypto/actions/cargo-cache@master
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@master
```

Comment:
Is there a reason for using `@master` instead of `@v1.0`?

Reply:
No particular reason, just force of habit. I basically just followed the example in the readme of `dtolnay/toolchain` which uses `@master`, too: https://github.com/dtolnay/rust-toolchain/tree/e12eda571dc9a5ee5d58eecf4738ec291c66f295#toolchain-expressions
Do you want me to replace it with `@v1` instead?

Reply:
Ah, it looks like `@v1` is not properly supported by the action: https://github.com/dtolnay/rust-toolchain/issues/45